src/components/layout/Header.js

```diff
@@ -17,10 +17,10 @@ const Header = () => (
     <Search className="header__search" />
     <nav className="header__nav">{nav}</nav>
     <nav className="header__social">
-      <a href="https://twitter.com/ApiPlatform" target="blank">
+      <a href="https://twitter.com/ApiPlatform" target="blank" aria-label="find us on twitter">
```

Comment:
should be `_blank`, not `blank` right? And so we must add `rel="noopener noreferrer"`
Same for other similar links